NEWS.rst

```diff
@@ -8,18 +8,11 @@ guake
 Release Summary
 ---------------
 
-Fix system font application issue
-
-Add fallback for version number finding
-
-Deprecated pbr
-
-Use keycodes instead of keyvals for keybindings. This allow to use keybindings with different keyboard layouts.
-
+This new version of Guake tries to catch up with
```

Comment:
Why is most of the release summary section deleted?

Reply:
ok i have fixed this glitch